tests/data/arxiv_api_response.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <link href="http://arxiv.org/api/query?search_query%3D%26id_list%3Dq-bio%2F0401024v1%2Cq-bio%2F0401014v1%2C1808.02949v2%26start%3D0%26max_results%3D400" rel="self" type="application/atom+xml"/>
+  <title type="html">ArXiv Query: search_query=&amp;id_list=q-bio/0401024v1,q-bio/0401014v1,1808.02949v2&amp;start=0&amp;max_results=400</title>
+  <id>http://arxiv.org/api/A+9fPrGUuO1OscO0gT4EFcXzHVM</id>
+  <updated>2022-01-10T00:00:00-05:00</updated>
+  <opensearch:totalResults xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">3</opensearch:totalResults>
+  <opensearch:startIndex xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">0</opensearch:startIndex>
+  <opensearch:itemsPerPage xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">400</opensearch:itemsPerPage>
+  <entry>
+    <id>http://arxiv.org/abs/q-bio/0401024v1</id>
+    <updated>2004-01-16T23:12:38Z</updated>
+    <published>2004-01-16T23:12:38Z</published>
+    <title>Morphogenesis by coupled regulatory networks</title>
+    <summary>  Based on a recently proposed non-equilibrium mechanism for spatial pattern
+formation [cond-mat/0312366] we study how morphogenesis can be controlled by
+locally coupled discrete dynamical networks, similar to gene regulation
+networks of cells in a developing multicellular organism. As an example we
+study the developmental problem of domain formation and proportion regulation
+in the presence of noise and cell flow. We find that networks that solve this
+task exhibit a hierarchical structure of information processing and are of
+similar complexity as developmental circuits of living cells. A further focus
+of this paper is a detailed study of noise-induced dynamics, which is a major
+ingredient of the control dynamics in the developmental network model. A master
+equation for domain boundary readjustments is formulated and solved for the
+continuum limit. Evidence for a first order phase transition in equilibrium
+domain size at vanishing noise is given by finite size scaling. A second order
+phase transition at increased cell flow is studied in a mean field
+approximation. Finally, we discuss potential applications.
+</summary>
+    <author>
+      <name>Thimo Rohlf</name>
+    </author>
+    <author>
+      <name>Stefan Bornholdt</name>
+    </author>
+    <arxiv:comment xmlns:arxiv="http://arxiv.org/schemas/atom">13 pages revTex, 17 figures (2 color)</arxiv:comment>
+    <link href="http://arxiv.org/abs/q-bio/0401024v1" rel="alternate" type="text/html"/>
+    <link title="pdf" href="http://arxiv.org/pdf/q-bio/0401024v1" rel="related" type="application/pdf"/>
+    <arxiv:primary_category xmlns:arxiv="http://arxiv.org/schemas/atom" term="q-bio.MN" scheme="http://arxiv.org/schemas/atom"/>
```

Comment:
So the `arxiv:primary_category` seems to be always equal to the first `catogory`, right? And that is why we don't extract the `arxiv:primary_category` in the code...

Are we sure it is always going to be the case?

Reply:
Yes, the `arxiv:primary_category` should always be also one of the `category` items, which is why it does not make sense to extract it a second time.


> seems to be always equal to the first `category`

I would not go as far as assuming anything on the order of the `<category>` elements, I think the primary category could also be not the first `<category>` element. But this should be irrelevant, unless we actually care about knowing which is the primary category.

Reply:
I see:) Thank you!